release-notes.html

```diff
@@ -26,7 +26,7 @@ <h3>Improvements</h3>
             <li>More in <a href="https://github.com/wso2/product-microgateway/issues?q=is%3Aissue+project%3Awso2%2Fproduct-microgateway%2F9+is%3Aclosed+label%3AType%2FImprovement">3.2.0 Improvements</a></li>
         </ul>
         <h3>Bug fixes</h3>
-        <p><a href="https://github.com/wso2/product-microgateway/issues?q=is%3Aissue+project%3Awso2%2Fproduct-microgateway%2F9+is%3Aclosed+label%3AType%2FBug">Fixed Issues</a></p>
+        <p><a href="https://github.com/wso2/product-microgateway/issues?q=is%3Aissue+project%3Awso2%2Fproduct-microgateway%2F9+is%3Aclosed+label%3AType%2FBug+milestone%3A3.2.0-RC1">Fixed Issues</a></p>
```

Comment:
This is not required. we need the whole set of bug fixed during entire 3.2.0 cycle. Not just the RC1. It looks like release note is already updated with correct content 👍 

```suggestion
        <p><a href="https://github.com/wso2/product-microgateway/issues?q=is%3Aissue+project%3Awso2%2Fproduct-microgateway%2F9+is%3Aclosed+label%3AType%2FBug+">Fixed Issues</a></p>
```